modules/oxide/src/less/theme/components/accessibility-checker/accessibility-checker.less

```diff
@@ -1,21 +1,22 @@
 //
 //
 // Accessibility checker
+@overlay-opacity: 80%;
 
 @accessibility-issue-info-heading-text-color: contrast(@dialog-background-color, @color-white, @color-tint);
 @accessibility-issue-info-body-text-color: contrast(@dialog-background-color, @color-white, @text-color);
-@accessibility-issue-info-background-color: contrast(@dialog-background-color, fade(darken(@color-tint, 3%), 10%), fade(darken(@color-tint, 3%), 40%));
+@accessibility-issue-info-background-color: mix(@background-color, @color-tint, @overlay-opacity);
 
 @accessibility-issue-warn-heading-text-color: contrast(@dialog-background-color, @color-white, darken(#cc8500, 12%));
 @accessibility-issue-warn-body-text-color: contrast(@dialog-background-color, @color-white, @text-color);
-@accessibility-issue-warn-background-color: contrast(@dialog-background-color, fade(orange, 8%), fade(orange, 50%));
+@accessibility-issue-warn-background-color: mix(@background-color, #FFCC00, @overlay-opacity);
```

Comment:
Hey Arvin, should we extract `#FFCC00` into its own variable like we did for `info` and `error`?

Reply:
done